pkg/alertmanager/multitenant.go

```diff
@@ -479,10 +479,46 @@ func (am *MultitenantAlertmanager) ServeHTTP(w http.ResponseWriter, req *http.Re
 	userAM, ok := am.alertmanagers[userID]
 	am.alertmanagersMtx.Unlock()
 
-	if !ok || !userAM.IsActive() {
+	if !ok && am.fallbackConfig != "" {
+		// Give the user the fallback configuration and start its alertmanager
+		userAmConfig, err := amconfig.Load(am.fallbackConfig)
```

Comment:
Instead of all such duplicated logic, couldn't we just call `setConfig()` with an empty `RawConfig`?